flink-tests/src/test/java/org/apache/flink/test/javaApiOperators/GroupCombineITCase.java

```diff
@@ -276,7 +276,7 @@ public void testPartialReduceWithDifferentInputOutputType() throws Exception {
 	// check if no shuffle is being executed
 	public void testCheckPartitionShuffleGroupBy() throws Exception {
 
-		org.junit.Assume.assumeTrue(mode != ExecutionMode.COLLECTION);
+		org.junit.Assume.assumeTrue(mode != TestExecutionMode.COLLECTION);
```

Comment:
I have committed this fix. If you rebase from master then you should get this change.